tutorials/evoked/plot_eeg_erp.py

```diff
@@ -190,7 +193,8 @@
 # If they are stored in a list, they can be easily averaged, for example,
 # for a grand average across subjects (or conditions).
 grand_average = mne.grand_average(all_evokeds)
-mne.write_evokeds('/tmp/tmp-ave.fif', all_evokeds)
+destination = op.join(tempfile.gettempdir(), 'tmp-ave.fif')
+mne.write_evokeds(destination, all_evokeds)
```

Comment:
Rather than actually write this, let's just comment it out. People can uncomment it if they want.
```suggestion
# And they can be written to disk like any other evoked data, e.g.:
# mne.write_evokeds('tmp-ave.fif', all_evokeds)
```